frontend/src/metabase/.eslintrc

```diff
@@ -6,7 +6,8 @@
         "patterns": [
           "metabase-enterprise",
           "metabase-enterprise/*",
-          "cljs/metabase.lib*"
+          "cljs/metabase.lib*",
+          "/embedding-sdk"
```

Comment:
If you're wondering why `/embedding-sdk` and not just `embedding-sdk`: https://metaboat.slack.com/archives/C505ZNNH4/p1736861240554919